src/pages/home/report/ReportActionsList.tsx

```diff
@@ -210,6 +211,10 @@ function ReportActionsList({
     );
     const prevSortedVisibleReportActionsObjects = usePrevious(sortedVisibleReportActionsObjects);
 
+    const reportLastReadTime = useMemo(() => {
+        return ReportConnection.getReport(report.reportID)?.lastReadTime ?? report.lastReadTime ?? '';
```

Comment:
This should never use `getReport()` from a lib. It should be `useOnyx` instead to connect to the report data.